src/main/java/com/odysseusinc/arachne/datanode/security/AuthenticationTokenFilter.java

```diff
@@ -64,15 +59,10 @@ public void doFilter(
         try {
 
             authenticationService.authenticate(authToken, httpRequest);
-            chain.doFilter(request, response);
         } catch (AuthenticationException | AuthException | org.ohdsi.authenticator.exception.AuthenticationException ex) {
-            ((HttpServletResponse) response).setStatus(HttpServletResponse.SC_UNAUTHORIZED);
-            JsonResult<Boolean> result = new JsonResult<>(JsonResult.ErrorCode.UNAUTHORIZED);
-            result.setResult(Boolean.FALSE);
-
-            response.getOutputStream().write(objectMapper.writeValueAsString(result).getBytes());
-            response.setContentType("application/json");
+            log.error("Authentication failed", ex);
```

Comment:
Do we really need full stacktrace here? Each time when a user logs in, makes long enough pause and goes back to Arachne (so goes to Arachne with invalid token), the exception will be logged, won't it?

Reply:
ok, sometimes stacktrace is useful to debug
let's leave full stacktrace in debug level only, is this approach acceptable?

Reply:
I think debug level makes good sense